src/tinode.js

```diff
@@ -5493,6 +5493,26 @@ TopicMe.prototype = Object.create(Topic.prototype, {
     writable: false
   },
 
+  contactsList: {
```

Comment:
BTW, this is pretty much the same as `contacts()`. Essentially you are adding the same method as `contacts` just under a different name.